.github/workflows/update_bank_branch_data.yml

```diff
@@ -3,9 +3,13 @@ name: Update bank/branch data
 on:
   schedule:
     - cron: "0 1 * * *" # 1am UTC
+  workflow_dispatch:
```

Comment:
Does this need to be present? Seems unusual to have an empty object.

Reply:
Just added this to allow us to trigger it manually through the github UI. Will help me test that this works too. From some examples I've seen this is how to enable this trigger when there are others. Will triple but happy to remove after we've tested if we'd prefer 👍